Prog/2019/Internet Access.sas

```diff
@@ -0,0 +1,183 @@
+/**************************************************************************
+ Program:  Internet Access.sas
+ Library:  Requests
+ Project:  NeighborhoodInfo DC
+ Author:   Yipeng Su
+ Created:  3/13/19
+ Version:  SAS 9.4
+ Environment:  Local Windows session
+ 
+ Description:  
+
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( ACS )
+%DCData_lib (RegHsg);
+
+%let year= 2013_17;
+
+proc format;
+  value Jurisdiction
```

Comment:
are you only looking at these inner jurisdictions?

Reply:
No, just used to flag inner jurisdictions